larkspur/larkspur.py

```diff
@@ -1,9 +1,10 @@
 import math
 import hashlib
 from struct import pack, unpack
+from typing import Any, List, Optional
```

Comment:
Import these types so you can replace the Any types used below

```suggestion
from typing import Any, List, Optional
from redis.client import Pipeline, Redis
```